circuit/src/main/kotlin/com/slack/circuit/Presenter.kt

```diff
@@ -62,7 +66,7 @@ interface Presenter<UiState, UiEvent : Any> where UiState : Any, UiState : Parce
    * }
    * ```
    */
-  @Composable fun present(render: StateRenderer<UiState, UiEvent>)
+  @Composable fun present(events: SharedFlow<UiEvent>): UiState
```

Comment:
We should only expose the `Flow` here, not the underlying type. We'll want to use an underlying channel like in my patch anyway so we're not waiting for consumers

```suggestion
  @Composable fun present(events: Flow<UiEvent>): UiState
```

Reply:
Done